user_interaction.py

```diff
@@ -1,110 +1,108 @@
-    os.system("gcloud auth login")
-    tensorboardCommand = "tensorboard --logdir=" + storingPath + " --port=8082"
-    os.system(tensorboardCommand)
+#Building the image
+build_command = "docker build -f Dockerfile -t " + image_URI + " ./"
+os.system(build_command)
+print("Docker image built")
+
+# Push the image on Google Cloud Registry
+pushing_image = "docker push " + image_URI
+os.system(pushing_image)
+print("Imaged pushed to Google Cloud Registry")
+
+print("Please insert a job name:")
+job_name = raw_input()
+
+region = raw_input("\nPlease insert the region you want to train \
+your model in or skip for the default value (europe-west4):")
+if region == "":
+    region = "europe-west4"
+
+batch_size = raw_input("\nPlease insert batch size\
+ or skip for the default value (128):")
+if batch_size == "":
+    batch_size = "128"
+
+no_of_steps = raw_input("\nPlease insert the number of steps for training\
+ or skip for the default value (10000):")
+if no_of_steps == "":
+    no_of_steps = "10000"
+
+steps_per_save = raw_input("\nPlease insert the number of steps per save\
+ or skip for the default value (300):")
+if steps_per_save == "":
+    steps_per_save = "300"
+
+steps_per_summary = raw_input("\nPlease insert the number of steps per summary\
+ or skip for the default value (300):")
+if steps_per_summary == "":
+    steps_per_summary = "300"
+
+no_of_checkpoints = raw_input("\nPlease insert the number of checkpoints to \
+keep or skip for the default value (10):")
+if no_of_checkpoints == "":
+    no_of_steps = "10"
+
+early_stop_loss_value = raw_input("\nPlease insert the early stop loss value\
+ or skip for the default value (5):")
+if early_stop_loss_value == "":
+    early_stop_loss_value = "5"
+
+learning_rate = raw_input("\nPlease insert the learning rate\
+ or skip for the default value (0.001):")
+if learning_rate == "":
+    learning_rate = "0.001" 
+
+# Submit the job on AI Platform
+print("Submitting the job on AI Platform")
+submitting_job = "gcloud beta ai-platform jobs submit training " + job_name\
++ " --region " + region + " --master-image-uri " + image_URI + " --config " + config_path + " -- --mode=train"\
++ " --save_dir=" + storing_path\
++ " --restore_dir=" + recovery_path\
++ " --gin_search_path=" + gin_path\
++ " --alsologtostderr"\
++ " --hp_tuning=True"\
++ " --gin_file=models/solo_instrument.gin"\
++ " --gin_file=datasets/tfrecord.gin"\
++ " --gin_param=\"TFRecordProvider.file_pattern='" + data_path\
++ "/train.tfrecord*'\""\
++ " --gin_param=checkpoints_to_keep=" + no_of_checkpoints\
++ " --gin_param=early_stop_loss_value=" + early_stop_loss_value\
++ " --gin_param=magenta_ddsp_internals.train_util.train.batch_size=" + batch_size\
++ " --gin_param=magenta_ddsp_internals.train_util.train.num_steps=" + no_of_steps\
++ " --gin_param=magenta_ddsp_internals.train_util.train.steps_per_summary=" + steps_per_summary\
++ " --gin_param=magenta_ddsp_internals.train_util.train.steps_per_save=" + steps_per_save
```

Comment:
I will add this super long flags support in the task.py script so the user will be able to use just short version, but I will keep you updated on it.